lib/Server.js

```diff
@@ -662,6 +663,13 @@ class Server {
       );
     }
 
+    if (!isWebTarget(this.compiler.options)) {
+      this.logger.info(`A non-web target was selected in dev server config.`);
+      if (this.options.liveReload) {
+        this.logger.info(`Live reload will not work with a non-web target.`);
+      }
+    }
```

Comment:
It is `warning`, also better keep it inside `DevServerPlugin`, in future `webpack-dev-server` will be plugin, so we need put logic inside plugin

Reply:
updated 👍🏻